app/v1/views/incident_view.py

```diff
@@ -0,0 +1,51 @@
+from flask import request, json, jsonify, Blueprint, make_response
+from flask_restful import Resource, Api, reqparse
+
+from app.v1.models.incidents import IncidentModel
+
+incident_api = Blueprint('incident_api',__name__)
+
+class IncidentApi(Resource):
+    """Shows a list of all incidents, and lets you post"""
+    def __init__(self):
+        self.incident = IncidentModel()
+
+    def get(self):
+        incidents = self.incident.get_all_incidents()
+        if incidents:
+            return make_response(jsonify(
```

Comment:
Choose whether to use make_response or jsonify and not both

Reply:
I used make _response to return json data and the relevant http status code as a param to make_response.
Is it fine that way?